src/main/java/com/google/devtools/build/lib/rules/proto/ProtoConfiguration.java

```diff
@@ -267,6 +284,24 @@ public StrictDepsMode strictProtoDeps() {
     return options.strictProtoDeps;
   }
 
+  @Override
+  public String starlarkStrictDeps() {
+    switch (strictProtoDeps()) {
+      case OFF:
+        return "OFF";
+
+      case WARN:
+        return "WARN";
```

Comment:
Do we actually use "WARN"? AFAIU it's only used for the Java sandwich, but that's passed in from Starlark. And if not,  how about using a Boolean?